xpholder/commands/mod/EditOtherCharacter.js

```diff
@@ -0,0 +1,134 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { sqlInjectionCheck, buildCharacterEmbed } = require("../../utils");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('edit__other_character')
+        .setDescription('Edit Another Player\'s Character [MOD]')
+        .addUserOption(option => option
+            .setName("player")
+            .setDescription("The Player You Wish To Edit")
+            .setRequired(true))
+        .addIntegerOption(option => option
+            .setName("character")
+            .setDescription("Which Character You Want To Edit ( 1 -> 10 )")
+            .setMinValue(1)
+            .setMaxValue(10)
+            .setAutocomplete(true)
```

Comment:
`.setAutocomplete(true)` should be removed here. It is causing an issue with the `client.autocomplete` call in main.js

Reply:
Done.